src/components/TheHeader.vue

```diff
@@ -96,9 +101,15 @@ const signupButtonLink = computed(() => {
 
 <style scoped>
 @media screen and (min-width: 960px) {
-  .desktop-logo {
+  .logo-container {
+    display: block;
```

Comment:
Because this is on the same line with the rest of the toolbar content, it would make more sense for this to have display: inline-block.

That said, because this element has `position: absolute`, it's removed from the document flow and the `display` property is not used at all. We shouldn't need this line
```suggestion
```

Reply:
Makes sense, will work on these changes